Project/Fall2020_CSC403_Project/FrmBattle.cs

```diff
@@ -70,7 +71,15 @@ public partial class FrmBattle : Form {
       }
 
       UpdateHealthBars();
-      if (player.Health <= 0 || enemy.Health <= 0) {
+      if (player.Health <= 0) {
```

Comment:
The player may also die while failing to flee. If you fetch changes from main, you can paste the logic from this statement in the flee button's function.